protobuf-matchers/BUILD

```diff
@@ -14,7 +14,6 @@
 # limitations under the License.
 #
 load("@rules_cc//cc:defs.bzl", "cc_library", "cc_proto_library", "cc_test")
-load("@rules_proto//proto:defs.bzl", "proto_library")
```

Comment:
Should we import it from https://github.com/protocolbuffers/protobuf/blob/dfd7784fa1147d3adcea304f1f414096e0bea765/bazel/proto_library.bzl#L14 then?

I think if you don't import it, we revert to using Bazel's native version. Isn't that discouraged?